### DISCUSSION TITLE
Could not find "org.jetbrains.kotlinx:kotlinx-coroutines-core"

### DISCUSSION THREAD

**Discussion by jdw** [korlibs/korge#2226] [General]
<img width="1512" alt="Screenshot 2024-05-06 at 18 57 08" src="https://github.com/korlibs/korge/assets/63590/640f1d41-d39e-4d12-bb3c-cfa1bc3a6533">

Full error message reads:
`Could not find "org.jetbrains.kotlinx:kotlinx-coroutines-core" in [/Users/VIP/dev/github.com/tobsef/Candy-Crush-Clone, /Users/VIP/.konan/klib, /Users/VIP/.konan/kotlin-native-prebuilt-macos-aarch64-1.7.20/klib/common, /Users/VIP/.konan/kotlin-native-prebuilt-macos-aarch64-1.7.20/klib/platform/macos_arm64]
`

Its the same for the dungeon, chess and cand crush examples, regretfully.

**Comment by soywiz** [comment]
compileKotlinMacosArm64 is not supported anymore, right now on desktop only the JVM target is supported. I guess that repo should be updated.
